src/js/index.js

```diff
@@ -0,0 +1,55 @@
+import Header from './header';
+import Home from './homepage';
+import About from './about';
+import Contact from './contact';
+import Menu from './menu';
+import Footer from './footer';
+
+const frag = document.createDocumentFragment();
+
+const mainSection = document.createElement('main');
+mainSection.className = 'main';
+mainSection.id = 'main';
+function main(section) {
+  return mainSection.appendChild(section);
+}
```

Comment:
```js
const main = section =>  mainSection.appendChild(section);
```

Consider refactoring to an arrow function

Reply:
Thanks for the review. I'll work on this.